operate/Makefile

```diff
@@ -99,6 +99,8 @@ start-e2e:
 	ZEEBE_BROKER_EXPORTERS_CAMUNDAEXPORTER_ARGS_CONNECT_INDEXPREFIX=e2eoperate \
 	ZEEBE_BROKER_EXPORTERS_CAMUNDAEXPORTER_ARGS_INDEX_SHOULDWAITFORIMPORTERS=false \
 	ZEEBE_BROKER_BACKPRESSURE_ENABLED=false \
+	ZEEBE_BROKER_NETWORK_COMMANDAPI_PORT=26505 \
+	ZEEBE_BROKER_NETWORK_INTERNALAPI_PORT=26504 \
```

Comment:
Can you explain why this is required? Is this something that every E2E test will need?

Reply:
The local E2E setup was broken so I used the opportunity to fix it